react/.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "vtex-react",
+  "extends": "vtex-react/io",
```

Comment:
what?

Reply:
Preset exclusively for io apps: https://github.com/vtex/typescript/blob/master/packages/eslint-config-vtex-react/io.js

The rationale behind it is to have separate rules for react native and io apps.

Reply:
who uses our rules in react native applications?

Reply:
Possibly instore related projects, but you're free to use the `vtex-react` one as well.

Reply:
strange